docs/guides/prometheus-grafana.md

```diff
@@ -0,0 +1,331 @@
+
+```bash
+POD_NAME=$(kubectl get pods --namespace monitoring -l "app=prometheus,component=server" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace monitoring  port-forward $POD_NAME 9090 &
+
+# You can access Prometheus in your browser at localhost:9090
+
+POD_NAME=$(kubectl get pods --namespace monitoring -l "app=grafana" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace monitoring port-forward $POD_NAME 3000 &
+
+# You can access Grafana in your browser at localhost:3000
+
+POD_NAME=$(kubectl get pods --namespace kong -l "app=kong" -o jsonpath="{.items[0].metadata.name}")
+kubectl --namespace kong port-forward $POD_NAME 8000 &
+
+# Kong proxy port is now your localhost 8000 port
+# We are using plain-text HTTP proxy for this purpose of
+# demo.
+# You can also use the LoadBalancer IP address and not set up this
+# port-forwarding if you are running in a cloud environment.
+```
+
+## Access Grafana Dashboard
+
+To access Grafana, you need to get the password for the admin user.
+
+Execute the following to read the password and take a note of it:
+
+```bash
+kubectl get secret --namespace monitoring grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+Now, browse to [http://localhost:3000](http://localhost:3000) and
+fill in username as “admin” and password as what you just noted above.
+You should be logged in to Grafana and Kong’s Grafana Dashboard
+should already be installed for you.
+
+## Setup Services
+
+We have all the components for monitoring installed,
+we will now spin up some services for demo purposes and setup Ingress
+routing for them.
+
+### Install Services
+
+We will setup three services: billing, invoice, comments.
+Execute the following to spin these services up:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/hbagdi/2d8ef66fe22cb99e1514f410f992268d/raw/a03d789b70c46ccd0b99d9f1ed838dc21419fc33/multiple-services.yaml
+```
+
+### Install Ingress for the Services
+
+Next, once the services are up and running, we will create Ingress
+routing rules in Kubernetes.
+This will configure Kong to proxy traffic destined for these services correctly.
+
+Execute the following:
+
+```bash
+echo "apiVersion: configuration.konghq.com/v1
+kind: KongIngress
+metadata:
+  name: strip-path
+route:
+  strip_path: true
+" | kubectl apply -f -
+
+echo "apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    configuration.konghq.com: strip-path
+  name: sample-ingresses
+spec:
+  rules:
+  - http:
+     paths:
+     - path: /billing
+       backend:
+         serviceName: billing
+         servicePort: 80
+     - path: /comments
+       backend:
+         serviceName: comments
+         servicePort: 80
+     - path: /invoice
+       backend:
+         serviceName: invoice
+         servicePort: 80" | kubectl apply -f -
+```
+
+## Let’s Create Some Traffic
+
+We’re done configuring our services and proxies.
+Time to see if our set up works or catches fire.
```

Comment:
```suggestion
Time to see if our setup works or catches fire.
```

Reply:
@hbagdi is there something more subtle we can say than "catches fire?" 

I think we can just say,

```suggestion
Time to see if our setup works.
```

Reply:
(going with the latter)